tools/amp_stt/kaldi_transcript_to_amp_transcript.py

```diff
@@ -11,8 +11,7 @@
 from speech_to_text import SpeechToText, SpeechToTextMedia, SpeechToTextResult, SpeechToTextScore, SpeechToTextWord
 
 # Convert kaldi output to standardized json
-def main():
-	(media_file, kaldi_file, kaldi_transcript_file, output_json_file) = sys.argv[1:5]
+def convert(media_file, kaldi_file, kaldi_transcript_file, output_json_file):
 	mgm_utils.exception_if_file_not_exist(kaldi_file)
 	mgm_utils.exception_if_file_not_exist(kaldi_transcript_file)
 	results = SpeechToTextResult()
```

Comment:
There is a change merged in master on line 17. try synch and see if any conflict.